examples/pxScene2d/src/pxScene2d.cpp

```diff
@@ -2942,9 +2956,9 @@ void pxScriptView::runScript()
 
     char buffer[1024];
 #ifdef RUNINMAIN
-    sprintf(buffer, "loadUrl(\"%s\");", url);
+    snprintf(buffer, sizeof(buffer), "loadUrl(\"%s\");", mUrl.cString());
 #else
-    sprintf(buffer, "loadUrl(\"%s\");", mUrl.cString());
+    snprintf(buffer, sizeof(buffer), "loadUrl(\"%s\");", mUrl.cString());
```

Comment:
If these are the same in both paths of the define, they should probably be moved outside of the define section.